cloud_storage_clients: introduce upstream registry

nvartolomei opened the pull request:
- stacks on https://github.com/redpanda-data/redpanda/pull/28896
- probably not the final form

<!--
See https://github.com/redpanda-data/redpanda/blob/dev/CONTRIBUTING.md#pull-request-body
for more details and examples of what is expected in a PR body.

Content in this top section is REQUIRED. Describe, in plain language, the motivation
behind the change (bug fix, feature, improvement) in this PR and how the included
commits address it.

Add the GitHub keyword `Fixes` to link to bug(s) this PR will fix, e.g.
  Fixes #ISSUE-NUMBER, Fixes #ISSUE-NUMBER, ...

If this PR is a backport, link to the original with `Backport of PR`, e.g.
  Backport of PR #PR-NUMBER
-->

## Backports Required

<!-- Checking at least one of the checkboxes is REQUIRED if this PR is not a backport. -->

- [x] none - not a bug fix
- [ ] none - this is a backport
- [ ] none - issue does not exist in previous branches
- [ ] none - papercut/not impactful enough to backport
- [ ] v25.3.x
- [ ] v25.2.x
- [ ] v25.1.x

## Release Notes
* none
<!--
If the changes in this PR do not need to be mentioned in the release
notes, then don't add a sub-section and simply list `none`, e.g.

* none

Otherwise, adding a sub-section or `none` is REQUIRED if the PR is not a backport PR.
If this is a backport PR, adding contents to this section will override
the release notes section inherited from the original PR to dev.

Add one or more of the sub-sections with a short description bullet
point of the change, e.g.

### Bug Fixes

* Short description of the bug fix if this is a PR to `dev` branch.

### Features

* Short description of the feature. Explain how to configure.

### Improvements

* Short description of how this PR improves existing behavior.

-->

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/77604-->
<!--eee8aecc-b050-4385-bf56-c34bf1943777-->
<!--dd61a166-ac9a-414d-83c3-4bbdbed1b7d3-->
<!--42209aa9-b128-472f-b980-b3fc4ca98293-->
<!--0a2233a6-8807-411d-a26c-d415d824e029-->
<!--764cdcb1-fe21-493b-adb7-1092a116661c-->
<!--01aca2fa-d6a5-4bea-a2a6-4db739e2471d-->
<!--155dd654-0047-4ce2-92c6-e7965971727b-->
<!--4e024f40-420d-4219-ae07-a9599fb2f39f-->
<!--title-->
## Retry command for [Build#77604](https://buildkite.com/redpanda/redpanda/builds/77604)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```
/ci-repeat 1
skip-redpanda-build
skip-units
skip-rebase
tests/rptest/tests/cluster_config_test.py::ClusterConfigTest.test_abs_cloud_validation@{"update_str":"ABS_ASK_OIDC_FEDERATION"}
tests/rptest/tests/cluster_config_test.py::ClusterConfigTest.test_abs_cloud_validation@{"update_str":"ABS_VM_INSTANCE_METADATA"}
tests/rptest/tests/cluster_config_test.py::ClusterConfigTest.test_abs_cloud_validation@{"update_str":"ABS_STATIC_CFG"}
tests/rptest/tests/cluster_config_test.py::ClusterConfigTest.test_cloud_validation
```
<!--/body-->

vbotbuildovich commented:
<!--TEST RESULTS-->
<!--0af87983-fd01-4669-9b8c-3a7f0690f456-->
<!--TEST RESULTS-->
<!--2b8b9182-6534-4dd8-a172-5fb4b54e8c5a-->
<!--title-->
## CI test results
<!--/title-->
<!--body-->
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/77604">build#77604</a></summary>


| test_class                | test_method               | test_arguments                             | test_kind   | job_url                                                                                   | test_status   | passed   | reason                                                                                                                                                                                                                                                                                                                                                                      | test_history                                                                                                                                              |
|:--------------------------|:--------------------------|:-------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------|
| ClusterConfigTest         | test_abs_cloud_validation | {"update_str": "ABS_ASK_OIDC_FEDERATION"}  | integration | https://buildkite.com/redpanda/redpanda/builds/77604#019b051a-6864-4ea0-bcd7-c4af7c33fdfb | FAIL          | 0/1      |                                                                                                                                                                                                                                                                                                                                                                             | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation |
| ClusterConfigTest         | test_abs_cloud_validation | {"update_str": "ABS_ASK_OIDC_FEDERATION"}  | integration | https://buildkite.com/redpanda/redpanda/builds/77604#019b0525-b9f8-4b7a-b72e-f156ced83c0f | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                            | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation |
| ClusterConfigTest         | test_abs_cloud_validation | {"update_str": "ABS_STATIC_CFG"}           | integration | https://buildkite.com/redpanda/redpanda/builds/77604#019b051a-6865-4309-a04a-a4948206ac44 | FAIL          | 0/1      |                                                                                                                                                                                                                                                                                                                                                                             | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation |
| ClusterConfigTest         | test_abs_cloud_validation | {"update_str": "ABS_STATIC_CFG"}           | integration | https://buildkite.com/redpanda/redpanda/builds/77604#019b0525-b9f9-424b-9513-1cf013d270ff | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                            | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation |
| ClusterConfigTest         | test_abs_cloud_validation | {"update_str": "ABS_VM_INSTANCE_METADATA"} | integration | https://buildkite.com/redpanda/redpanda/builds/77604#019b051a-6866-4c0f-8c84-6c1d8db88d58 | FAIL          | 0/1      |                                                                                                                                                                                                                                                                                                                                                                             | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation |
| ClusterConfigTest         | test_abs_cloud_validation | {"update_str": "ABS_VM_INSTANCE_METADATA"} | integration | https://buildkite.com/redpanda/redpanda/builds/77604#019b0525-b9fc-47fd-a856-3e91ae0632aa | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                            | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation |
| ClusterConfigTest         | test_cloud_validation     | null                                       | integration | https://buildkite.com/redpanda/redpanda/builds/77604#019b051a-686a-40ce-b76e-7d8b40f9c7ce | FAIL          | 0/1      | Test PASSES after retries.[Inconclusive result after max retries](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#Case-F%3A-inconclusive-result-after-max-retries-(run-passes)%5BinlineExtension%5D)(baseline=0.0000, p0=1.0000, reject_threshold=0.0100. adj_baseline=0.0000, p1=1.0000, trust_threshold=0.5000) | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_cloud_validation     |
| ClusterConfigTest         | test_cloud_validation     | null                                       | integration | https://buildkite.com/redpanda/redpanda/builds/77604#019b0525-ba02-49ac-8958-610b9492f4c7 | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                            | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_cloud_validation     |
| CertificateRevocationTest | test_kafka                | null                                       | integration | https://buildkite.com/redpanda/redpanda/builds/77604#019b051a-686a-40ce-b76e-7d8b40f9c7ce | FLAKY         | 50/51    | Test PASSES after retries.[Inconclusive result after max retries](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#Case-F%3A-inconclusive-result-after-max-retries-(run-passes)%5BinlineExtension%5D)(baseline=0.0000, p0=1.0000, reject_threshold=0.0100. adj_baseline=0.0000, p1=1.0000, trust_threshold=0.5000) | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=CertificateRevocationTest&test_method=test_kafka        |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/77898">build#77898</a></summary>


| test_class                       | test_method                                | test_arguments                                                               | test_kind   | job_url                                                                                   | test_status   | passed   | reason                                                                                                                                                                                                                                                                                                                                                                                              | test_history                                                                                                                                                                 |
|:---------------------------------|:-------------------------------------------|:-----------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| ClusterConfigTest                | test_abs_cloud_validation                  | {"update_str": "ABS_ASK_OIDC_FEDERATION"}                                    | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23f9-36ed-4ebb-8050-2ad10557fc15 | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation                    |
| ClusterConfigTest                | test_abs_cloud_validation                  | {"update_str": "ABS_ASK_OIDC_FEDERATION"}                                    | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23fd-2fa4-4eff-aa92-8d850b29e246 | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation                    |
| ClusterConfigTest                | test_abs_cloud_validation                  | {"update_str": "ABS_STATIC_CFG"}                                             | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23f9-36ee-4687-9e37-2d4f80b0ddd4 | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation                    |
| ClusterConfigTest                | test_abs_cloud_validation                  | {"update_str": "ABS_STATIC_CFG"}                                             | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23fd-2fa5-4c21-ad4d-e75d7f2f1d5d | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation                    |
| ClusterConfigTest                | test_abs_cloud_validation                  | {"update_str": "ABS_VM_INSTANCE_METADATA"}                                   | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23f9-36ee-4474-b5e0-83856294c048 | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation                    |
| ClusterConfigTest                | test_abs_cloud_validation                  | {"update_str": "ABS_VM_INSTANCE_METADATA"}                                   | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23fd-2fa7-4dd4-be11-146af8569f52 | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_abs_cloud_validation                    |
| ClusterConfigTest                | test_cloud_validation                      | null                                                                         | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23f9-36f0-4d7f-b4f7-af01598c93bd | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_cloud_validation                        |
| ClusterConfigTest                | test_cloud_validation                      | null                                                                         | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23fd-2fb1-4a85-8542-318892031334 | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=ClusterConfigTest&test_method=test_cloud_validation                        |
| CrashLoopChecksTest              | test_cloud_storage_client_misconfiguration | null                                                                         | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23f9-36ed-4903-9506-c66adb404b56 | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=CrashLoopChecksTest&test_method=test_cloud_storage_client_misconfiguration |
| CrashLoopChecksTest              | test_cloud_storage_client_misconfiguration | null                                                                         | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23fd-2fa3-4416-982c-4bd4ce5508a6 | FAIL          | 0/11     | Test FAILS after retries.[Significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-B%3A-Test-fails-with-an-improbably-higher-rate-than-baseline-(run-fails))(baseline=0.0000, p0=0.0000, reject_threshold=0.0100)                                                                    | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=CrashLoopChecksTest&test_method=test_cloud_storage_client_misconfiguration |
| Datalake3rdPartyMaintenanceTest  | test_e2e_basic                             | {"catalog_type": "nessie", "cloud_storage_type": 1, "query_engine": "spark"} | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23f9-36ee-4474-b5e0-83856294c048 | FLAKY         | 50/51    | Test PASSES after retries.[Inconclusive result after max retries](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#Case-F%3A-inconclusive-result-after-max-retries-(run-passes)%5BinlineExtension%5D)(baseline=0.0004, p0=1.0000, reject_threshold=0.0100. adj_baseline=0.0013, p1=0.9358, trust_threshold=0.5000)                         | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=Datalake3rdPartyMaintenanceTest&test_method=test_e2e_basic                 |
| MountUnmountIcebergTest          | test_simple_remount                        | {"cloud_storage_type": 1}                                                    | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23fd-2fb5-475a-bec7-139ac7a74a0e | FLAKY         | 9/11     | Test PASSES after retries.[No significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-C%3A-Test-passes-with-a-rate-not-significantly-worse-than-baseline-(run-passes))(baseline=0.1977, p0=0.8895, reject_threshold=0.0100. adj_baseline=0.4836, p1=0.0140, trust_threshold=0.5000) | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=MountUnmountIcebergTest&test_method=test_simple_remount                    |
| WriteCachingFailureInjectionTest | test_unavoidable_data_loss                 | null                                                                         | integration | https://buildkite.com/redpanda/redpanda/builds/77898#019b23fd-2faf-4a73-9d8e-93c8b8cbccb3 | FLAKY         | 35/41    | Test PASSES after retries.[No significant increase in flaky rate](https://redpandadata.atlassian.net/wiki/spaces/DEVPROD/pages/1419247661/Statistical+retry+failure+reasons#%5BinlineExtension%5DCase-C%3A-Test-passes-with-a-rate-not-significantly-worse-than-baseline-(run-passes))(baseline=0.0595, p0=0.0872, reject_threshold=0.0100. adj_baseline=0.1681, p1=0.3153, trust_threshold=0.5000) | https://redpanda.metabaseapp.com/dashboard/87-tests?tab=142-dt-individual-test-history&test_class=WriteCachingFailureInjectionTest&test_method=test_unavoidable_data_loss    |
</details>
<!--/body-->

nvartolomei committed:
cloud_storage_clients: introduce upstream registry

nvartolomei committed:
cloud_storage_clients: distinguish bucket_name (DSN) from plain_bucket_name

Introduce type distinction between bucket_name and plain_bucket_name:

- bucket_name: Can store either a plain bucket name OR a DSN (Data Source
  Name) with connection parameters (e.g., "my-bucket?region=us-west-2").
  This type was introduced to support cross-region bucket access for Remote
  Read Replicas.

- plain_bucket_name: Always contains just the bucket name without any
  parameters (e.g., "my-bucket"). This is the actual bucket name used for
  cloud storage API calls.

Also introduce bucket_params struct to hold extracted plain_bucket_name and
future connection parameters parsed from bucket_name DSN format.

This type safety prevents accidentally using a DSN string where only the
plain bucket name is expected, and makes the distinction explicit in client
interfaces.

nvartolomei committed:
cloud_storage_clients: rename http_client_ptr to client_ptr

Less confusing name.

nvartolomei committed:
cloud_storage_clients: more descriptive counting methods

nvartolomei committed:
cloud_storage_clients: rename _pool to _idle_list

nvartolomei committed:
cloud_storage_clients: lazy client pool initialization

nvartolomei committed:
cloud_storage_clients: rework client tracking in the client pool

nvartolomei commented:
/ci-repeat 1

nvartolomei commented:
/ci-repeat 1

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/77898-->
<!--b79ca6aa-4426-4f7a-b4d7-8bf151919c12-->
<!--d052b30d-f3af-44c3-a80d-6ff0cb58a654-->
<!--c6d03027-d6da-410a-8aef-a93d7b68d291-->
<!--5e4ffef4-fe0d-4b61-a596-698d8afe8e82-->
<!--46ee411e-3c45-4d25-b136-b0ec30ec9a04-->
<!--d9581e9a-9c45-4c3b-9ff0-45726dfbf80e-->
<!--b85a038d-4ddf-48ea-b209-18b9cca1b664-->
<!--a9dca4f2-1259-470d-bb5d-e2c398e1581a-->
<!--b4e14599-e06a-4dab-895f-e75fbc6a52e5-->
<!--0d70cd9f-40d4-40aa-879e-c5b5ee11dc01-->
<!--title-->
## Retry command for [Build#77898](https://buildkite.com/redpanda/redpanda/builds/77898)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```
/ci-repeat 1
skip-redpanda-build
skip-units
skip-rebase
tests/rptest/tests/cluster_config_test.py::ClusterConfigTest.test_abs_cloud_validation@{"update_str":"ABS_ASK_OIDC_FEDERATION"}
tests/rptest/tests/crash_loop_checks_test.py::CrashLoopChecksTest.test_cloud_storage_client_misconfiguration
tests/rptest/tests/cluster_config_test.py::ClusterConfigTest.test_abs_cloud_validation@{"update_str":"ABS_STATIC_CFG"}
tests/rptest/tests/cluster_config_test.py::ClusterConfigTest.test_cloud_validation
tests/rptest/tests/cluster_config_test.py::ClusterConfigTest.test_abs_cloud_validation@{"update_str":"ABS_VM_INSTANCE_METADATA"}
```
<!--/body-->

nvartolomei committed:
wi

nvartolomei commented:
* https://github.com/redpanda-data/redpanda/pull/29100
* https://github.com/redpanda-data/redpanda/pull/29410
